### PR TITLE
[JSX] Added Checkbox Element to Form.js

### PR DESCRIPTION
**Pull request by HenriRabalais**
A react component called `Checkbox` Element was added to Form.js. This can be used in any reactified form in LORIS.

Testing instructions coming soon...

### CODE REVIEW

File: jsx/Form.js [hunk 1]
```diff
@@ -1285,6 +1285,95 @@ LinkElement.defaultProps = {
...
+      requiredHTML = <span className="text-danger">*</span>;
+    }
+
+    // Add error message
+    if (this.props.errorMessage) {
+      errorMessage = <span>{this.props.errorMessage}</span>;
+      elementClass = 'row form-group has-error';
+    }
+
+    if (this.props.label) {
```
Comment (HenriRabalais) [inline jsx/Form.js]: This was introduced for potential in-line formatting, in the case where there was no label. However, it is not necessary and label can be made a mandatory field.

Comment (zaliqarosli) [reply jsx/Form.js]: i agree with label not needing to be a mandatory field

File: jsx/Form.js [hunk 2]
```diff
@@ -1378,6 +1467,9 @@ class LorisElement extends React.Component {
       case 'link':
         elementHtml = (<LinkElement {...elementProps} />);
         break;
+      case 'checkbox':
```
Comment (zaliqarosli) [inline jsx/Form.js]: case 'advcheckbox'

File: jsx/Form.js [hunk 3]
```diff
@@ -1285,6 +1285,95 @@ LinkElement.defaultProps = {
...
+    }
+
+    return (
+      <div className={elementClass}>
+        {label}
+        <div className={this.props.inputClass}>
+          <input
+            type="checkbox"
+            className="form-control"
+            style={{width: '5%'}}
```
Comment (zaliqarosli) [inline jsx/Form.js]: we should remove this and make a class with css instead

File: jsx/Form.js [hunk 4]
```diff
@@ -1285,6 +1285,95 @@ LinkElement.defaultProps = {
...
+      );
+    }
+
+    return (
+      <div className={elementClass}>
+        {label}
+        <div className={this.props.inputClass}>
+          <input
+            type="checkbox"
+            className="form-control"
```
Comment (zaliqarosli) [inline jsx/Form.js]: remove form-control and add input-sm

**Comment by zaliqarosli** [comment]
@driusan changed to an approving review!

**Pull request merged**